src/ol_infrastructure/applications/mitxonline/__main__.py

```diff
@@ -162,4 +163,14 @@
 )
 mitxonline_vault_backend = OLVaultDatabaseBackend(mitxonline_vault_backend_config)
 
+# Set Consul key for use in edxapp configuration template
+consul.Keys(
+    "mitxonline-app-domain-for-edxapp",
+    keys=[
+        consul.KeysKeyArgs(
+            path="edxapp/marketing-domain", value=mitxonline_config.require("domain")
```

Comment:
Doesn't this also need the edxapp/mfe-domain?

Reply:
No, because that's getting set in the edxapp project.